organization/principles/en.md

```diff
@@ -0,0 +1,183 @@
+- Margaritis George, Member, managing PyGreece community outreach.
+- Nikolaou Lysandros Nikolaou, Administrator.
+- Petkos Theofanis, Administrator.
+- Skias Panagiotis, managing PyGreece community projects.
+
+### Board Lifecycle
+
+Each board serves an annual term. In order to elect a new board a General Assembly (GA)
+needs to be hosted. The General Assembly needs to be called at least 15 days before the
+preferred date. At the same point, 15 days before the GA, the board needs to select an
+independent election committee & to find all the next board of directors candidates.
+
+### Inactivity of Directors
+
+In case a board member is inactive (has not attended 4 board meetings in a row) the board
+has the right to vote for their replacement. They are able to select another member as
+long as this member is a confirmed PyGreece community member.
+
+### Re-Election Limit
+
+A member can serve a specific role (e.g Administrator or Responsible for PyCon Greece) two
+times in a row maximum. A member can be elected as a board member for a maximum of six
+times. The limits are reset in case the member takes a break for a term from the role
+and/or the board membership.
+
+### Board Elections
+
+Only registered members of the community have voting rights. The board is responsible to
+communicate a registration form through the PyGreece Discord Server 2 weeks before the
+elections.
+
+The election committee, selected by the previous board, has no right to participate in the
+elections. The committee:
+
+- Hosts the vote to confirm that the financial status of the organization is acceptable by
+  the GA & to end the previous board term.
+- Decides on the voting process that outlines the format of the Directors elections.
+- Decides beforehand the process of resolving ties.
+- Hosts the vote for the next board of directors.
+
+Post election, the newly elected board decides the roles as they are described in the
+board structure section.
+
+## PyGreece Community Principles
+
+Below you can find the main principles of PyGreece community, the Greek Python community
+managed by the PyGreece AMKE. The byLaws of PyGreece AMKE entity are different and can be
+found [here](https://publicity.businessportal.gr/company/180308907000).
+
+> [!IMPORTANT] The board of PyGreece AMKE is currently working on the legal format of the
+> community principles mentioned below. Therefore, while they are considered a very
+> important part of the way that PyGreece community operates, they don't have any form
+> legal force yet.
+
+- The name of the community is "PyGreece Community", abbreviated "PyGreece".
+
+- The name of the legal entity responsible for "PyGreece Community" is "PyGreece AMKE".
+
+- PyGreece AMKE is a civil society actor (Civil Non-Profit Company) created by Greek
+  Python community members to serve the mission of PyGreece community. The mission of
+  PyGreece AMKE can be found [here](../about/en.md#mission).
```

Comment:
```suggestion
  PyGreece AMKE can be found [on our mission document](../about/en.md#mission).
```

Reply:
maybe only "on this document" sounds better here?

Reply:
The rationale here is that [acessibility is best when the link texts include some kind of information about the link target](https://webaim.org/techniques/hypertext/#link_to_link).

Reply:
Yeah no strong opinion. Let's go with that.